pdocs/templates/text.mako

```diff
@@ -30,7 +52,19 @@ ${func.docstring}
 ```python3
 ${var.name}
 ```
+% if hasattr(var, 'parsed_docstring') and var.parsed_docstring:
+    # table with arguments info
+    % if var.parsed_docstring.params:
+| Parameter | type | description | default |
+|---|---|---|---|
+        % for p in var.parsed_docstring.params:
+| ${p.arg_name} | ${p.type_name} | ${p.description} | ${p.default} |
+        % endfor
+    % endif
```

Comment:
We're rendering a single variable here, no need for a table.
also, I don't think we have a default or arg_name here, only a description and type.